E: Version '89*' for 'chromium-browser' was not found

Fala Julio, tudo certo? Primeiramente, muito obrigado por disponibilizar a imagem!

Tentei fazer o build, e recebi ```E: Version '89*' for 'chromium-browser' was not found```. Ao que parece esta versão do chromium não está mais disponível para ```ubuntu:18.04```.  Não fiz mais nenhum teste.

Segue mensagem completa: 

```
E: Version '89*' for 'chromium-browser' was not found 
The command '/bin/sh -c apt-get update && apt-get install -y     libnss3-tools     zenity     libgtk2.0-0     dbus-x11     yad     libcurl3     libdbus-1.3     sudo gosu     libxss1     lsb-release     wget     xdg-utils     net-tools     openssl=${OPENSSL_VERSION}*     xfce4=${XFCE_VERSION}*     xfwm4=${XFWM4_VERSION}*     x11vnc=${X11VNC_VERSION}*     xvfb=${XVFB_VERSION}*     novnc=${NOVNC_VERSION}*     chromium-browser=${CHROME_VERSION}*     && dpkg -l | awk '{print "|" $2 "|" $3 "|"}' > /installed-packages.txt' returned a non-zero code: 100
make: *** [Makefile:14: build] Error 10
```
Valeu!!!

--- a/Dockerfile
+++ b/Dockerfile
@@ -7,7 +7,7 @@ RUN apt-get update && \
 FROM ubuntu:18.04
 
 ENV DEBIAN_FRONTEND=noninteractive \
-    CHROME_VERSION=89 \
+    CHROME_VERSION=91 \
     XFCE_VERSION=4.12.4 \
     XFWM4_VERSION=4.12.5 \
     X11VNC_VERSION=0.9.13-3 \

